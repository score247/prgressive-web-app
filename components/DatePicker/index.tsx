import "react-datepicker/src/stylesheets/datepicker.scss";
import "./style.scss";

import React from 'react';
import DatePicker, { ReactDatePickerProps } from "react-datepicker";

export default React.memo(function(props: ReactDatePickerProps) {
    return <DatePicker {...props} />;
});
