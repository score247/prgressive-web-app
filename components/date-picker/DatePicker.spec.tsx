import React from "react";
import { shallow } from "enzyme";
import DatePicker from "./index";
import { ReactDatePickerProps } from "react-datepicker";

describe("DatePicker", () => {
  it("DatePicker-render correctly", () => {
    const props: ReactDatePickerProps = {
      selected: new Date(),
      onChange: function(date: Date) {}
    };

    const wrapper = shallow(<DatePicker {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
