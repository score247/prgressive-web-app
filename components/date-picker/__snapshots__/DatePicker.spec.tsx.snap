// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker DatePicker-render correctly 1`] = `ShallowWrapper {}`;
